packages/components/src/hooks/use-hover.ts

```diff
@@ -4,6 +4,7 @@ export const useHover = <T extends HTMLElement & SVGSVGElement>(
     refSetter?: RefObject<T> | null,
     should_prevent_bubbling?: boolean
 ) => {
+    const isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
```

Comment:
is it possible to use already defined `isMobile` from stores?

Reply:
We can't import isMobile from stores since the components package doesn't have any stores hook implemented inside

Reply:
I think you can use `IsMobile` from shared package for the time being till components package is exposed to stores

Reply:
Reuse `isMobileOs` from the shared package.